backend/tests/v5/unit/middleware/dataConverter/inputs/teamspaces/projects/models/commons/groups.test.js

```diff
@@ -79,6 +80,13 @@ const deserialiseGroup = (group) => {
 		}
 	}
 
+	if (output?.rules) {
+		output.rules = output.rules.map((r) => ({
```

Comment:
Since you don't want to copy the field, split it with the spread operator and you can address it directly as well.
```suggestion
		output.rules = output.rules.map(({field, ...rest}) => ({
		   ...rest
```